lib/phoenix/controller.ex

```diff
@@ -128,6 +128,12 @@ defmodule Phoenix.Controller do
   def endpoint_module(conn), do: conn.private.phoenix_endpoint
 
   @doc """
+  Returns the socket handler module as an atom, raises if unavailable.
+  """
+  @spec socket_handler_module(Plug.Conn.t) :: atom
+  def socket_handler_module(conn), do: conn.private.phoenix_socket_handler
```

Comment:
Oh sorry, I mean both Transport.WebSocket and Transport.LongPoller use it. We can remove and reference the priv in both places though?